src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/IRemoteAuthenticationProvider.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.VisualStudio.Composition;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Debug
+{
+    [ProjectSystemContract(ProjectSystemContractScope.UnconfiguredProject, ProjectSystemContractProvider.Private, Cardinality = ImportCardinality.ZeroOrMore)]
+    public interface IRemoteAuthenticationProvider
```

Comment:
Given there are no other implementations of this, can we make this internal for now until we need to add to it?